src/test/scala/firrtlTests/transforms/DedupTests.scala

```diff
@@ -254,7 +254,7 @@ class DedupModuleTests extends HighTransformSpec {
      execute(diff_params, diff_params, Seq.empty)
   }
 
-  "Modules with aggregate ports that are bulk connected" should "NOT dedup if their port names differ" in {
+  "Modules with aggregate ports that are (partial)? connected" should "NOT dedup if their port names differ" in {
```

Comment:
Both `<-` and `<=` of aggregates are a bulk connect no? At least that's what I was thining

Reply:
I was pedantically viewing this as `<-` is "partial connect" and `<=` is "connect" and `chisel2.<>` is "bulk connect". It's trivial, I can change it.

Reply:
Yeah fair, I'm not sure if my terminology is really the most popular, but that's how I've been using it